package.json

```diff
@@ -71,7 +71,8 @@
     "@cumulus/api": "19.2.0-alpha.1",
     "@cumulus/aws-client": "19.2.0-alpha.1",
     "@cypress/webpack-preprocessor": "^5.11.0",
-    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
+    "@testing-library/jest-dom": "^6.5.0",
```

Comment:
Where do we need these @testing-library/ libraries?

Reply:
Yeah, we haven't moved to Jest yet. We are still Ava for unit tests. I think it may have been one of those get all the dependencies when you at RTL and this wasn't noticed when that happened.

Reply:
I removed as part of the amend PR